sdk/src/main/scala/com/horizen/account/state/ForgerStakeMsgProcessor.scala

```diff
@@ -249,10 +268,12 @@ case class ForgerStakeMsgProcessor(params: NetworkParams) extends FakeSmartContr
     }
 
     // TODO decide whether we need to check also genesis case (also UTXO model)
```

Comment:
this todo can be removed: it was decided to be ok.

Reply:
Ok, done